src/operation-node/operation-node-transformer.ts

```diff
@@ -390,6 +390,8 @@ export class OperationNodeTransformer {
       temporary: node.temporary,
       ifNotExists: node.ifNotExists,
       onCommit: node.onCommit,
+      frontModifiers: node.frontModifiers,
```

Comment:
You need to transform these `this.transformNodeList(node.frontModifiers)`

Reply:
Done!